### PR TITLE
Replaced period 2 spaces with period 1 space, fix a few obvious misspellings part of #11984

### PR DESCRIPTION
**Pull request by paul1956**
Fixed a few obvious spelling errors
These changes should only should ONLY be in comments, there are still 34 places in code and many in Resource files not fixed

Fixes part of #11984
REVIEWERS - there should be NO changes outside of comments.

## Proposed changes
Fix all instances of ".  " in C# and VB comments ONLY

## Customer Impact

- None

## Regression? 

-No

## Risk
 - None, only comments were changes

## Test methodology

- No code changes

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12028)

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
@paul1956  - looks like your regular expression removed the 2 spaces following a period rather than replaced them with a single space. It would be easier to review if the non-white space changes(spelling) were in a separate PR.

**Comment by paul1956** [comment]
@Tanya-Solyanik it that happened it should have been very few places; I will review again and correct any places that happened. Otherwise do I need to start again I don't know how to undo the spelling corrections.

**Comment by paul1956** [comment]
> @paul1956 - looks like your regular expression removed the 2 spaces following a period rather than replaced them with a single space. It would be easier to review if the non-white space changes(spelling) were in a separate PR.

@Tanya-Solyanik There were 8 files incorrectly changes, most in 1 project they are now fixed, I have rereviewed each file individually. There are just a few spelling correction mostly to avoid Merge issues because they are also being fixed simultaneously,  "behaviour" is partly changed, but it should be changed everywhere to behavior.

File: src/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace System.Drawing;
 ///     See the ImageInfo.cs file for the helper nested ImageInfo class.
 ///
 ///     A common pattern for using this class is as follows (See PictureBox control):
-///     1. The winform app (user's code) calls ImageAnimator.Animate() from the main thread.
+///     1. The WinForm app (user's code) calls ImageAnimator.Animate() from the main thread.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs:14]:
```suggestion
///     1. The WinForms app (user's code) calls ImageAnimator.Animate() from the main thread.
```

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/SelectionManager.cs [hunk 2]
```diff
@@ -8,25 +8,25 @@
...
     private BehaviorService _behaviorService;           // ptr back to our BehaviorService
     private IServiceProvider _serviceProvider;          // standard service provider
     private readonly Dictionary<IComponent, ControlDesigner> _componentToDesigner;    // used for quick look up of designers related to components
     private readonly Control _rootComponent;            // root component being designed
-    private ISelectionService _selectionService;                  // we cache the selection service for perf.
-    private IDesignerHost _designerHost;                // we cache the designerhost for perf.
-    private Rectangle[]? _previousSelectionBounds;           // used to only repaint the changing part of the selection
-    private object? _previousPrimarySelection;               // used to check if the primary selection changed
+    private ISelectionService _selectionService;        // we cache the selection service for perf.
+    private IDesignerHost _designerHost;                // we cache the designerHost for perf.
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/SelectionManager.cs:20]:
```suggestion
    private IDesignerHost _designerHost;                // we cache the DesignerHost for perf.
```
alternatively we can remove most of these comments and replace the useful ones(lines 21,22,25) with comments on the line above the field definition. If you choose to re-do comments please do it in a new PR.

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DesignerToolStripControlHost.cs [hunk 3]
```diff
@@ -8,7 +8,7 @@
 namespace System.Windows.Forms.Design;
 
 /// <summary>
-///  This internal class is used by the new ToolStripDesigner to add a dummy node to the end. This class inherits from ToolStripControlHost and overrides the  CanSelect property so that the dummy Node when shown in the designer doesnt show selection on Mouse movements. The image is set to theDummyNodeImage embedded into the resources.
+///  This internal class is used by the new ToolStripDesigner to add a dummy node to the end. This class inherits from ToolStripControlHost and overrides the  CanSelect property so that the dummy Node when shown in the designer doesn't show selection on Mouse movements. The image is set to theDummyNodeImage embedded into the resources.
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DesignerToolStripControlHost.cs:11]: You still have 2 spaces before CanSelect. Consider breaking this comment in shorter lines if you touch it.

File: src/System.Windows.Forms/src/System/Windows/Forms/Layout/TableLayout.cs [hunk 4]
```diff
@@ -294,9 +294,9 @@ private static void AssignRowsAndColumns(ContainerInfo containerInfo)
             while (!xAssignRowsAndColumns(containerInfo, childrenInfo, estimatedCols, numRows, growStyle))
             {
                 // I am assuming that the division will put us pretty close to the right
-                // number of columns.  If this assumption is wrong, a binary search
-                // between the number we get by dividing and childrenInfos.Count
-                // could be more efficient.  It would certainly degenerate better.
+                // number of columns. If this assumption is wrong, a binary search
+                // between the number we get by dividing and childrenInfo's.Count
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Layout/TableLayout.cs:298]:
```suggestion
                // between the number we get by dividing and `childrenInfo.Count`
```

**Review by Tanya-Solyanik — COMMENTED** [review]
I reviewed non-whitespace changes and glanced through the whitespace ones. I left a couple of comments on the spelling.

File: src/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs (hunk 1 above)
Comment (paul1956) [reply src/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs:14]: Fixed all of them

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/SelectionManager.cs (hunk 2 above)
Comment (paul1956) [reply src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/SelectionManager.cs:20]: I will undo change and move up on a new PR. that just fixes spelling issues. Moving comment up is probably a separate issue.

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DesignerToolStripControlHost.cs (hunk 3 above)
Comment (paul1956) [reply src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/DesignerToolStripControlHost.cs:11]: I will be doing a separate PR to shorten comments after this is merged due to merge conflicts but will fix this one here.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you!

**Pull request merged**